app/services/request_create_service.rb

```diff
@@ -82,7 +91,7 @@ def auto_approve(request)
 
     sub_requests = request.parent? ? request.children : [request]
 
-    sub_requests.reverse.each do |req|
+    sub_requests.reverse_each do |req|
```

Comment:
I thought `reverse_each` does not for a relation

Reply:
It's by mistake, changed by rubocop.